src/protocol/requests/initProducerId/v0/response.js

```diff
@@ -0,0 +1,37 @@
+const Decoder = require('../../../decoder')
+const { failure, createErrorFromCode, failIfVersionNotSupported } = require('../../../error')
+
+/**
+ * InitProducerId Response (Version: 0) => throttle_time_ms error_code producer_id producer_epoch
+ *   throttle_time_ms => INT32
+ *   error_code => INT16
+ *   producer_id => INT64
+ *   producer_epoch => INT16
+ */
+const decode = async rawData => {
+  const decoder = new Decoder(rawData)
+  const throttleTime = decoder.readInt32()
+  const errorCode = decoder.readInt16()
+
+  failIfVersionNotSupported(errorCode)
+
+  return {
+    throttleTime,
+    errorCode,
+    producerId: decoder.readInt64(),
```

Comment:
```suggestion
    producerId: decoder.readInt64().toString(),
```

It's better to keep the int64 as a string to avoid any coupling with `Long`, we plan to replace `Long` with the new BigInt. It's mainly used for operations.

Reply:
Done!